src/data/markdown/docs/01 guides/05 Testing Guides/04 Large-scale k6 testing.md

```diff
@@ -0,0 +1,484 @@
+---
+title: "Hardware-sizing for large-scale k6 tests"
+excerpt: "How to run large-scale k6 tests without distributed-execution"
+---
+
+This document explains how to launch a large-scale k6 test on a single machine without the need of distributed execution. 
+
+The common misconception of many load-testers is that distributed-execution (ability to launch a load test on multiple machines) is required to generate large load. This is not the case with k6.
+
+k6 is different from many other load-testing tools in a way it handles hardware resources. A single k6 process will efficiently use all CPU cores on a load-generator machine.
+Single instance of k6 is often enough to generate load of 30.000-40.000 simultaneus users (VUs). This amount of VUs can generate upwards of 300.000 requests per second (RPS). 
+
+Unless you need more than 100.000-300.000 requests per second (6-12M requests per minute), a single instance of k6 will likely be sufficient for your needs.
+
+Below we will explore what hardware is needed for generating different levels of load.
+
+
+<div class="doc-blockquote">
+
+> ### Note about native distributed-execution in k6
+>
+> The long-term goal for k6 is to support distributed execution natively. We are currently laying groundwork for this feature in [PR #1007](https://github.com/loadimpact/k6/pull/1007).
+> You can follow this effort on guthub [in issue 140](https://github.com/loadimpact/k6/issues/140). Again, this is useful only if you need to generate load larger than 300k RPS.
```

Comment:
```suggestion
> You can follow this effort on GitHub [in issue 140](https://github.com/loadimpact/k6/issues/140). Again, this is useful only if you need to generate load larger than 300k RPS.
```
:smile:

Reply:
I think we should move to guthub, sounds awesome!